show-changes

```diff
@@ -14,25 +14,30 @@ fi
 
 OLD="$1"
 NEW="${2-HEAD}"
+OLD_FMT=""
 
+if [[ $OLD == *"lts"* ]]; then
+  OLD_FMT=$(echo $OLD | tr "-" " " | tr '[:lower:]' '[:upper:]')
+else
+  OLD_FMT=$(echo $OLD | tr "-" " " | sed 's/./\U&/')
+fi
```

Comment:
Maybe this block could get a comment about what this thing is about. I had to run it in the terminal myself to understand it.

Reply:
Ack!